### ISSUE TITLE
spacemacs: eldoc error

### ISSUE THREAD

**Issue by intuxicated** [xcwen/ac-php#66]
I have just installed the `phpplus` layer but in any php file this error will be shown
```
 REBUILD:  rebuild file start
eldoc error: (wrong-type-argument listp ac-php-phptags-index-process-filter)
ERROR ----- 
```
any idea how I can solve this problem?

---

```
$ php -v
PHP 7.2.0 (cli) (built: Dec  3 2017 21:46:44) ( NTS )
Copyright (c) 1997-2017 The PHP Group
Zend Engine v3.2.0, Copyright (c) 1998-2017 Zend Technologies
    with Xdebug v2.6.0alpha1, Copyright (c) 2002-2017, by Derick Rethans
```
```
$ phpctags --version
Version: 0.5.1
Exuberant Ctags compatiable PHP enhancement, Copyright (C) 2012 Techlive Zheng
Addresses: <techlivezheng@gmail.com>, https://github.com/techlivezheng/phpctags
```
```
$ cscope --version
cscope: version 15.8b
```
```lisp
dotspacemacs-configuration-layers
   '(
     phpplus
     ;; ----------------------------------------------------------------
     ;; Example of useful layers you may want to use right away.
     ;; Uncomment some layer names and press <SPC f e R> (Vim style) or
     ;; <M-m f e R> (Emacs style) to install them.
     ;; ----------------------------------------------------------------
     helm
     ;; auto-completion
     ;; better-defaults
     emacs-lisp
     ;; git
     ;; markdown
     ;; org
     ;; (shell :variables
     ;;        shell-default-height 30
     ;;        shell-default-position 'bottom)
     ;; spell-checking
     ;; syntax-checking
     ;; version-control
     )
```

**Comment by xcwen** [comment]
now ac-php jion in spacemace (develep) you can checkout develep spacemacs  and test it

**Comment by intuxicated** [comment]
same error in develop branch.

**Comment by xcwen** [comment]
show emacs version

**Comment by xcwen** [comment]
you don't need install phpctags . 
don't set  ac-php-ctags-executable  with other value
(defvar ac-php-ctags-executable (concat   ac-php-root-directory "phpctags"))

**Comment by dcostaras** [comment]
I'm getting the same error:

GNU Emacs 27.0.50 (build 1, x86_64-apple-darwin17.3.0, NS appkit-1561.20 Version 10.13.2 (Build 17C88)) of 2018-01-10                                                                                                                                                                                             Spacemacs develop: 0.300.0@27.0.50 (spacemacs)

It only happens on one project, other projects seem to work fine. Is there more information about the error reported somewhere?

**Comment by xcwen** [comment]
show me the example project :>

**Comment by xcwen** [comment]
retest .

**Comment by dcostaras** [comment]
Hi @xcwen ,

I've moved away from develop branch so as far as I know it's not using ac-php. I remember solving this problem though, it was an issue with the project. I found this by recreating the project from scratch and copy pasting across from the old project. It would be super helpful if the error was a bit more descriptive, like were the syntax error is occurring.

Thanks,
Look forward to when this drops in spacemacs stable
